Add Support section to README (#310)
woodpecker-ci/woodpecker: README.md

![woodpecker](docs/docs/woodpecker.png)

## Support

> Please consider to donate to your open-collective 🙏 [[Become a backer](https://opencollective.com/woodpecker-ci#category-CONTRIBUTE)]

Reviewer: ```suggestion
Please consider to donate and become a backer. 🙏 [[Become a backer](https://opencollective.com/woodpecker-ci#category-CONTRIBUTE)]
```

> Please considder to download to your opencollective 🙏 [[Become a backer](https://opencollective.com/woodpecker-ci#category-CONTRIBUTE)]

Author: ```suggestion
Please consider to donate to your open-collective 🙏 [[Become a backer](https://opencollective.com/woodpecker-ci#category-CONTRIBUTE)]
```
